### DISCUSSION TITLE
What does x-vercel-ip-country return for browsers in Puerto Rico?

### DISCUSSION THREAD

**Discussion by LL782** [vercel/community#6823] [Help]
### Summary

Does "x-vercel-ip-country" return `PR` or `US` for browsers in Puerto Rico?

### Steps to Reproduce

Our checkout is setup with Puerto Rico as a state of US, which I'm not able to reconfigure. 

We use 'x-vercel-ip-country' to get a country code and look-up prices based that code. As we don't have prices for PR I'm trying to understand if browsers from PR will be getting prices or not. 

If I knew what value they get back from 'x-vercel-ip-country' that would be useful information

**Comment by versecafe** [comment]
@LL782 Based off Vercel Analytics Puerto Rico is flagged as a separate country, which likely means it is for `x-vercel-ip-country` but no guarantees without being able to test from a device in Puerto Rico

<img width="1334" alt="image" src="https://github.com/vercel/community/assets/147033096/ba9ec066-af17-42d8-997a-af3c60582152">

**Reply by LL782** [reply]
That's very informative, thank you.
